### ISSUE TITLE
[Proposal] Reflect size of specific wasm binary on a wheel

### ISSUE THREAD

**Issue by MaxGraey** [boyanio/wasm-wheel#9]

**Comment by boyanio** [comment]
Hey Max,

Thank you for the proposal. Most of wheel parts (languages) compile directly to wasm and we could put the file size on the wheel part. Languages like C# and PHP, however, work as interpreters in the wheel, i.e. the interpreter is compiled to wasm and the source file is given as a parameter. What do you think we should do in these scenarios?

Best,
Boyan

**Comment by MaxGraey** [comment]
I think that it will be difficult for such languages ​​to separate part of the CLR from runtime and the actual code, and you just need to specify the final size, considering the CLR as part of the runtime.

**Comment by boyanio** [comment]
Good point. Maybe for C# one could write "X bytes CRL + X bytes code" with text or something similar. I could try to play around with the styling to see if I could manage to squeeze it on the wheel part :-)
